Assets/Scripts/Room/Chat/ChatCommandHandler.cs

```diff
@@ -0,0 +1,722 @@
+    /// Set field of view
+    /// </summary>
+    /// <param name="input"></param>
+    private static void SetFov(string input)
+    {
+        int fov;
+        if (int.TryParse(input, out fov))
+        {
+            Camera.main.fieldOfView = fov;
+            var message = $"Field of vision set to {fov}.";
+            instance.chatRoom.AddMessage(message);
+        }
+        else
+        {
+            instance.chatRoom.OutputErrorMessage("Fov has to be a number");
+        }
+    }
+
+    private static void ToggleSpecMode()
+    {
+        settings[0xf5] = (int)settings[0xf5] == 1 ? 0 : 1;
+        bool specMode = (int)settings[0xf5] == 1;
+        instance.EnterSpecMode(specMode);
+        string message = FormatSystemMessage(specMode ? "You have entered spectator mode." : "You have exited spectator mode.");
+        instance.chatRoom.AddMessage(message);
+    }
+
+    /// <summary>
+    /// Restart current game
+    /// </summary>
+    private static void RestartGame()
+    {
+        if (isMasterClient)
+        {
+            var chatMessage = new object[] { FormatSystemMessage("MasterClient has restarted the game!"), string.Empty };
+            instance.photonView.RPC("Chat", PhotonTargets.All, chatMessage);
+            instance.restartRC();
+        }
+        else
+        {
+            instance.chatRoom.OutputErrorNotMasterClient();
+        }
+    }
+
+    private static void SendPrivateMessage(string[] parameters)
+    {
+        int playerId;
+        var playerIdString = parameters[1];
+        if (int.TryParse(playerIdString, out playerId))
+        {
+            var targetPlayer = PhotonPlayer.Find(playerId);
+
+            StringBuilder chatMessage = new StringBuilder();
+            for (var messageIndex = 2; messageIndex < parameters.Length; messageIndex++)
+            {
+                chatMessage.Append(parameters[messageIndex] + " ");
+            }
+            instance.photonView.RPC("ChatPM", targetPlayer, new object[] { SetNameColorDependingOnteam(player), chatMessage.ToString() });
+
+            var message = $"TO [{targetPlayer.ID}] {SetNameColorDependingOnteam(targetPlayer)}:{chatMessage}";
+            instance.chatRoom.AddMessage(message);
+        }
+        else
+        {
+            instance.chatRoom.OutputErrorPlayerNotFound(playerIdString);
+        }      
+    }
+
+    /// <summary>
+    /// Reset your kd
+    /// </summary>
+    private static void ResetKd()
+    {
+        var hashTable = new Hashtable();
+        hashTable.Add(PhotonPlayerProperty.kills, 0);
+        hashTable.Add(PhotonPlayerProperty.deaths, 0);
+        hashTable.Add(PhotonPlayerProperty.max_dmg, 0);
+        hashTable.Add(PhotonPlayerProperty.total_dmg, 0);
+        player.SetCustomProperties(hashTable);
+
+        var message = "Your stats have been reset.";
+        instance.chatRoom.AddMessage(message);
+    }
+
+    private static void ChangeRoomProperties(string[] input)
+    {
+        if (isMasterClient)
```

Comment:
There's several if statements here. Code can be made more readable if
```
if (!isMasterClient)
{
    instance.chatRoom.OutputErrorNotMasterClient();
    return;
}

// Now we can assume that we are the master client
```

The remaining code has to be in one less if statement.